cg/meta/create_validation_cases/validation_cases_api.py

```diff
@@ -0,0 +1,175 @@
+"""Module that holds the api to create validation cases."""
+
+import logging
+from pathlib import Path
+
+from housekeeper.store.models import File, Version
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.constants import SequencingFileTag
+from cg.models.validation_cases.validation_case_data import ValidationCaseData
+from cg.store.models import Case, CaseSample, Sample
+from cg.store.store import Store
+from cg.utils.files import copy_file, get_files_matching_pattern, rename_file
+
+LOG = logging.getLogger(__name__)
+
+
+class ValidationCaseAPI:
```

Comment:
```suggestion
class CreateValidationCaseAPI:
```

Or are you leaving the option for this API to also perform the validation in the future?